src/pages/userPage/ActiveMembership/ActiveMembership.tsx

```diff
@@ -31,7 +31,7 @@ const ActiveMembership = () => {
   const [data, setUserData] = useState<Data>();
   const [currentUser, setCurrentUser] = useState<any>({});
   const [LoadInfo, setLoadInfo] = useState<boolean>(false);
-  const [plastDegrees, setPlastDegrees] = useState<Array<UserPlastDegree>>([]);
+  const [plastDegrees, setPlastDegrees] = useState<UserPlastDegree>({} as UserPlastDegree);
```

Comment:
the name "plastDegrees" is not correct now, because it only stores 1 degree, so maybe rename it

Reply:
Totally agree

Reply:
Done